python/cudf/cudf/core/_base_index.py

```diff
@@ -1444,7 +1444,8 @@ def drop_duplicates(
         """
 
         # This utilizes the fact that all `Index` is also a `Frame`.
-        result = self.__class__._from_columns(
+        # Except RangeIndex.
```

Comment:
Yeah, we'll probably want to move this impl eventually. No specific urgency though.

Reply:
Let's introduce this in this round, and move it in the next cycle.